SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -200,25 +200,10 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 		}
 	}
 
-	const bool bTrackHandoverProperties = ShouldTrackHandoverProperties();
```

Comment:
Still reading - but previously we could prevent handover properties from replicating via this check. I assume that's not possible anymore in the current setup?

Reply:
ie. stop server_only properties replicating when they aren't needed

Reply:
yeah, thought about this a bit, we could reimplement this, but wasn't sure how much it's needed.